src/test/java/org/mockitousage/debugging/VerificationListenerCallBackTest.java

```diff
@@ -12,30 +11,22 @@
 import org.mockito.verification.VerificationEvent;
 import org.mockito.verification.VerificationMode;
 
-
 import java.lang.reflect.Method;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.Assert.assertNull;
 import static org.junit.Assert.fail;
 import static org.mockito.Mockito.*;
 
 public class VerificationListenerCallBackTest {
-    private Method invocationWanted;
-    private RememberingListener listener;
-    private MockitoFramework mockitoFramework;
-
-    @Before
-    public void setUp() throws NoSuchMethodException {
```

Comment:
Then it is just my personal preference to do it otherwise, but I am okay with this approach so lets leave it.